crates/ty_python_semantic/src/types/visitor.rs

```diff
@@ -156,7 +156,9 @@ impl<'db> From<Type<'db>> for TypeKind<'db> {
             | Type::ModuleLiteral(_)
             | Type::ClassLiteral(_)
             | Type::SpecialForm(_)
-            | Type::Dynamic(_) => TypeKind::Atomic,
+            | Type::Dynamic(_)
+            // REVIEWERS: is this correct?
+            | Type::NewTypeInstance(_) => TypeKind::Atomic,
```

Comment:
ummm you could probably argue it both ways. I think this is fine for now; if we find a reason in the future why it isn't fine we can always change it